lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -396,9 +714,9 @@ sub job_restart {
     }
 
     # if we got new jobs, notify workers
-    if (@duplicated) {
-        notify_workers;
-    }
+    # if (@duplicated) {
+    #     notify_workers;
+    # }
```

Comment:
thanks for the input - i think we can even safely drop notify_workers at all in the rest of the codebase